service/awsconfig/v5/resource/loadbalancer/current.go

```diff
@@ -2,9 +2,87 @@ package loadbalancer
 
 import (
 	"context"
+
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/aws-operator/service/awsconfig/v5/key"
+)
+
+const (
+	cloudProviderClusterTagValue = "owned"
+	cloudProviderServiceTagKey   = "kubernetes.io/service-name"
 )
 
-// TODO
 func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interface{}, error) {
-	return nil, nil
+	customObject, err := key.ToCustomObject(obj)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	currentState, err := r.clusterLoadBalancers(customObject)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	return currentState, nil
+}
+
+func (r *Resource) clusterLoadBalancers(customObject v1alpha1.AWSConfig) (*LoadBalancerState, error) {
+	lbState := &LoadBalancerState{}
+
+	// We get all load balancers because the API does not allow tag filters.
+	output, err := r.clients.ELB.DescribeLoadBalancers(&elb.DescribeLoadBalancersInput{})
+	if err != nil {
+		return lbState, microerror.Mask(err)
```

Comment:
Lets please simply return nil. Anything else will just lead to problems in the long run.

Reply:
Ack. Sorry I missed this. Now done.